docs/user/_sidebar.md

```diff
@@ -1,5 +1,7 @@
 * [Home](/README.md)
 * [Kyma module](README.md)
+* [Module Lifecycle](01-manager.md)
+* [Module Configuration](02-configuration.md)
```

Comment:
These should be identical to the docs' titles. Please use sentence case for titles.